### DISCUSSION TITLE
NextJS + ConnectKit + Wagmi v2 + CustomChain = Chain not configured.  Version: @wagmi/core@2.10.2

### DISCUSSION THREAD

**Discussion by I0San** [wevm/wagmi#3944] [Q&A]
Hi,

I am wondering if anyone else is experiencing problems with latest versions of wagmi/viem/connectKit?

When adding a custom chain (used the avalanche example with type Chain from Viem), the switch network with Metamask is failing and also for certain hooks (useLatestBlock) I am getting the "Chain not configured" error.

Just wondering. Can provide more details and maybe an example app if needed.

Thanks.

**Comment by humblemodulo** [comment]
I'm having the same problem.  Did you ever make any progress on this?  The interesting thing is, I can make use of the standard `wagmi` `use` hooks and things like `writeContract` work fine.  However, when attempting to make use of `wagmi/core` "direct" calls, I receive the "Chain Not Configured" errors.
